### ISSUE TITLE
Color contrast issues

### ISSUE THREAD

**Issue by robdekort** [laravel/prompts#192]
### Laravel Prompts Version

0.3.5

### Laravel Version

12.17.0

### PHP Version

8.2.28

### Operating System & Version

macOS 15.5

### Terminal Application

iTerm

### Description

I've noticed questions that have been answered and unselected items in prompts like multisearch fail in color contrast. You can see that very well in these examples:

<img width="1171" alt="Image" src="https://github.com/user-attachments/assets/31a24e40-b7bc-4f03-911b-82f16b8f070c" />

<img width="825" alt="Image" src="https://github.com/user-attachments/assets/67cdeeba-43a6-4a25-8d30-355c1908f022" />

### Steps To Reproduce

.
